here-naksha-lib-psql/src/jvmMain/resources/pako.sql

```diff
@@ -6,21 +6,35 @@ DO $sql$ BEGIN
     EXECUTE $outer$
       CREATE OR REPLACE FUNCTION gzip(uncompressed BYTEA, compression_level INTEGER) RETURNS BYTEA AS $inner$
         if (!globalThis["require"]) plv8.find_function('es_modules_init')();
-        return require('pako').deflate(uncompressed, {"level":compression_level});
+        return require('pako').gzip(uncompressed, {"level":compression_level});
```

Comment:
Please notice that I changed wrapper from `deflate` to `gzip`.
According to benchmarks https://nodeca.github.io/pako/#Deflate
![image](https://github.com/user-attachments/assets/8771ee7d-abb5-4de9-8e5c-78812e3699dd)

gzip is slower than deflate, especially in older node.js version, but imo we should not use deflate inside of gzip method as it leads to bugs - the java coda has to use appropriate method to decompress it.

We can optimize it be adding a flag telling us which wrapper to use, or by introducing wrapper auto-detection (for gzip - verification of 2 first bytes), but it should go with methods renaming.